odml/tools/rdf_converter.py

```diff
@@ -189,11 +204,17 @@ def get_rdf_str(self, rdf_format):
                Full lists see in odml.tools.format_converter.FormatConverter._conversion_formats
         :return: string object
         """
+        if rdf_format not in self._conversion_formats:
+            raise ValueError("odml.RDFWriter.get_rdf_str: Format for output files is incorrect. "
+                             "Please choose from the list: {}".format(self._conversion_formats.keys()))
         return self.convert_to_rdf().serialize(format=rdf_format).decode("utf-8")
 
-    def write_file(self, filename, rdf_format):
+    def write_file(self, filename, rdf_format="turtle"):
         data = self.get_rdf_str(rdf_format)
-        with open(filename, "w") as file:
+        filename_ext = filename
+        if not filename.find("." + rdf_format):
```

Comment:
Here it probably needs to check for `cls._conversion_formats.get(rdf_format)` to properly check and add the file extension.